databases/week1/homework.sql

```diff
@@ -0,0 +1,75 @@
+-- ques1 : Find out how many tasks are in the task table
+select count(id)
+from task;
+
+
+-- ques2 : Find out how many tasks in the task table do not have a valid due date
+
+select count(title)
```

Comment:
why did you use title?
are you sure the result is correct? if so, why?
;)

Reply:
I can count id as well, actually id is a good idea it will take less space then title. And i think the result is correct because i counted for only null which is not a due date, and i assume except null everything is valid

Reply:
what if a task had a NULL as title? ;)